Tabbing mechanism was overwriting every anchor

pjmckennawork opened the pull request:
Tabbing mechanism was overwriting every anchor, not just the ones it was meant to.

Changed the selector to work on class, not tag type.

pjmckenna-ibm committed:
Tabbing mechanism was overwriting every anchor, not just the ones it was meant to.
Changed the selector to work on class, not tag type.

pjmckenna-ibm committed:
Fix for problem with Console. The Console will always send a header to the server indicating IE7.
The override of the IE rendering engine version using the meta tag
<meta http-equiv="X-UA-Compatible" content="IE=9">
has no effect on the useragent header sent by the browser to the server, but does change the documentMode.

pjmckennawork commented:
Added a second commit related to properly dealing with angular.js incorrectly determining the requested IE version.

eoinjoyc commented:
The tabbing mechanism fix should be ok to add.

Having a look at the latest angular.js code, it looks like this fix has been added. 
https://github.com/angular/angular.js/blob/master/src/Angular.js#L191
We should upgrade the angular version in the adf, rather than modifying a third party library. Angular v1.3.8 has been stable for us in our project, and it contains the fix.

Can you revert the angular.js change, and I can accept the request?

pjmckenna-ibm committed:
Latest version of ADF includes a version of Angular with this fix.
Revert "Fix for problem with Console. The Console will always send a header to the server indicating IE7."

This reverts commit ecbb39166933cd0811a9402eaa6a0983ab27ce67.

pjmckennawork commented:
I have reverted the last commit, leaving just the Tabbing fix.
